pkg/util/resource.go

```diff
@@ -44,6 +44,26 @@ func MultiplyQuant(quant resource.Quantity, factor float64) resource.Quantity {
 	return *newQuant
 }
 
+// Min returns the result of Min(a, b) for each named resource, corresponded to the quotav1.Max().
+func Min(a corev1.ResourceList, b corev1.ResourceList) corev1.ResourceList {
```

Comment:
The semantics of this function are a bit strange, such as A is {cpu = 1}, B is {memory = 1}, after Min is {cpu = 1, memory = 1}. For quotav1.Max, union keys is semantic, but min is a bit problematic.

Reply:
Fixed. Now the unmatched resource names will be dropped in the result.
e.g. a = {"cpu": "1"}, b = {"memory", "2Gi"} => {}